.travis.yml

```diff
@@ -1,3 +1,8 @@
 language: node_js
 node_js:
-  - "6.9"
+  - "12.8.0"
+script: bash ./deploy.sh
+env:
+    global:
+    - ENCRYPTION_LABEL: "a668d2a97a80"
+    - COMMIT_AUTHOR_EMAIL: "mbrad012@ucr.edu"
```

Comment:
Whenever you finish doing research please remind me to change this to my email

Reply:
I can change it to yours now if you prefer. No promises that I'll remember in a month or so.

Reply:
Yeah you should probably change it now. Then I'll have to remember to update it when I leave (although there might not be anyone to update it to)

Reply:
It's been changed.